Sources/Extension/Decimal+Extension.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Decimal+Extension.swift
+//  MusicNotationCore
+//
+//  Created by Bartłomiej Świerad on 05/11/2019.
+//
+
+import Foundation
+
+extension Decimal {
```

Comment:
We shouldn’t use Decimal; can we just use Double? And then when you need an Int, just use the Int initializer? We don’t want to import Foundation.

Reply:
Importing Foundation was just my mistake because it is imported by default in a new file. We can use Double though if it's enough accurate